docker-compose.yml

```diff
@@ -23,5 +35,7 @@ services:
             PGDATABASE: leapchat
```

Comment:
@jimmcgaw How about we make `PGPORT` 5433 so those of us running Postgres on our host can still run LeapChat for local dev?

Reply:
Want to do this?

Reply:
I think we can leave PGPORT as 5432, but bind it to 5433 on the host, so the container doesn't conflict.

The default 5432 should still be usable by other containers in the network.

Reply:
Sounds good